### DISCUSSION TITLE
How to join the development of this project

### DISCUSSION THREAD

**Discussion by weihongx9315** [sandboxie-plus/Sandboxie#1734] [General]
I fixed some bugs of sandboxie and try some new features,such as fix the cred part of sandboxie(when i use mstsc,it can't save my password),hook DnsQuery api to let app use the specified dns server,fix some iWebOffice plugin with browser and so on.I want to join this project,thx.

**Comment by DavidXanatos** [comment]
For starters you can submit a pull request with your changes.

**Reply by weihongx9315** [reply]
Ok,thanks

**Reply by DavidXanatos** [reply]
so whats the ETA on the pull request ;)

**Reply by weihongx9315** [reply]
Sorry for being a little busy these days, because there are some additional functional modifications, but they are not within the scope of the bug, so I am thinking about whether to extract it separately and submit it. If I can't extract this part of the code recently, I will push the part of bug fix first.
Any question，please let me know.
 
Best regards,
